recipes/snakemake/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - docutils
     - pyyaml
     - requests >=2.8.1
-    - dropbox >=5.2
```

Comment:
Keep in mind this will not create a new snakemake package as you did not increased the build-number.
I think this is ok, as it will not break anything and the next update of snakemake will get the new package dependency, just wanted to make you aware.